### PR TITLE
Install help2man

### PR DESCRIPTION
**Pull request by josegonzalez**

### CODE REVIEW

**Comment by michaelshobbs** [comment]
:+1:

File: Makefile [hunk 1]
```diff
@@ -34,6 +34,7 @@ copyfiles: addman
 		done
 
 addman:
+	apt-get install -qq -y help2man
```
Comment (josegonzalez) [inline Makefile]: Should we add help2man target here as well?

File: Makefile [hunk 2]
```diff
@@ -23,7 +23,7 @@ all:
 
 install: dependencies stack copyfiles plugin-dependencies plugins version
 
-copyfiles: addman
```
Comment (josegonzalez) [inline Makefile]: Why?

Comment (michaelshobbs) [inline Makefile]: This told make to run `addman` before `copyfiles`. `help2man` calls dokku and thus needs to be in the `$PATH` first

**Comment by michaelshobbs** [comment]
Removed the redundant call to install help2man. I think help2man makes sense to be called in the addman task.

**Pull request merged**